cmd/buck/cli/util.go

```diff
@@ -37,8 +38,15 @@ func handleProgressBars(p *uiprogress.Progress, events chan local.PathEvent) {
 	for e := range events {
 		switch e.Type {
 		case local.FileStart:
+			if runtime.GOOS == "windows" {
+				cmd.Message("Uploading %s", e.Path)
+				continue
+			}
 			bars[e.Path] = addBar(p, e.Path, e.Size)
 		case local.FileProgress, local.FileComplete:
+			if runtime.GOOS == "windows" {
+				continue
+			}
```

Comment:
We could be creative here and print some dots on progress, but that might create a weird output if the file is too big and the screen gets full of dots. Another option is crafting a way of always printing the % of progress but I'd guess that requires some careful deletion of already printed values and re-printing; sounds like something a library should do.

Sounds to me that we should explore eventually an `uiprogress` similar component that works well in all OSs, instead of jumping in building our own custom Windows solution.

Reply:
I haven't used it, but this golang based UI toolkit looks pretty nice https://github.com/charmbracelet/bubbletea

Reply:
Nice, sounds like a good candidate :)

Reply:
Cool, that looks nice! We've made a [number of custom tweaks](https://github.com/textileio/uiprogress/commits/master) to the `uiprogress` package. Maybe bubbletea supports the things that we're needed. I think the main one is the ability to replace each bar with a final state, e.g., the pushed / pulled Cid. Will take a look!